content/templates/content/artisanprogram.html

```diff
@@ -0,0 +1,51 @@
+{% extends "content/base.html" %}
+
+{% block content %}
+
+ {% for program in artisanprograms %}
+ {% if forloop.counter0|divisibleby:2 %}
+ 	
+  {% if forloop.first %}
+ <div class="row">
+ 	{% if program.img %} <!--  if image 4, 8 col -->
+  	<div class="col-md-4">
+		<h1 class="childrenstitle text-center text-uppercase">{{program.title}}</h1>
+	</div>
+	<div class="col-md-8">
+	 <img class="center-block img-responsive programimg" src="{{program.img.url}}" alt="Children Studying">
```

Comment:
Not 100% sure that you should set the alt without knowing what the image is but its not that important. feel free to ignore :)